gum/backend-darwin/gumdarwinmoduleresolver.c

```diff
@@ -318,7 +318,7 @@ gum_darwin_module_resolver_rebuild_indexes (GumDarwinModuleResolver * self,
     mod = g_ptr_array_index (latest_modules, i);
     path = gum_module_get_path (mod);
 
-    if (g_str_has_suffix (path, "/usr/lib/dyld_sim"))
```

Comment:
Note: `dyld_sim` will not be included unless `gum_collect_modules` falls back to `gum_collect_modules_forensically` branch. So this branch was dead code